akka-http/src/main/scala/akka/http/scaladsl/common/NameReceptacle.scala

```diff
@@ -16,19 +17,35 @@ class NameReceptacle[T](val name: String) {
   def as[B] = new NameReceptacle[B](name)
   def as[B](unmarshaller: Unmarshaller[T, B])(implicit fsu: FSU[T]) =
     new NameUnmarshallerReceptacle(name, fsu.transform[B](implicit ec => implicit mat => { _ flatMap unmarshaller.apply }))
+  /** Symbolic alias for [[optional]]. */
   def ? = new NameOptionReceptacle[T](name)
+  def optional = new NameOptionReceptacle[T](name)
+  /** Symbolic alias for [[withDefault]]. */
   def ?[B](default: B) = new NameDefaultReceptacle(name, default)
+  def withDefault[B](default: B) = new NameDefaultReceptacle(name, default)
```

Comment:
```suggestion
  def orElse[B](default: B) = new NameDefaultReceptacle(name, default)
```
How about `orElse`? I'm not entirely sure I like it more than `withDefault`, TBH.